lightbeam/api.py

```diff
@@ -84,7 +121,7 @@ def get_retry_client(self):
             retry_options=ExponentialRetry(
                 attempts=self.lightbeam.config['connection']["num_retries"],
                 factor=self.lightbeam.config['connection']["backoff_factor"],
-                statuses=self.lightbeam.config['connection']["retry_statuses"]
+                statuses=self.lightbeam.config['connection']["retry_statuses"].append(401)
```

Comment:
Could the 401 retry status be included by default in the config, instead of appending it here?

Reply:
Possibly, although then if the user provides custom `retry_statuses` but _doesn't_ include 401, `lightbeam` would fail as soon as it needs to re-auth... do we want to let a user encounter that situation?